examples/README.md

```diff
@@ -702,7 +702,7 @@ In order to test the Metrics feature locally in a [Kind](https://kind.sigs.k8s.i
 ## Storage Drivers
 
 Beside filesystem storage backend, zot also supports S3 storage backend, check below url to see how to configure it:
-- [s3 config](https://github.com/docker/docker.github.io/blob/master/registry/storage-drivers/s3.md): A driver storing objects in an Amazon Simple Storage Service (S3) bucket.
+- [s3 config](https://github.com/docker/docker.github.io/blob/master/registry/storage-drivers/s3.md): A driver storing objects in an Amazon Simple Storage Service (S3) bucket. **_[ NO DOCUMENTATION --- DEAD LINK ] MAYBE THIS ONE BUT NO EXPLICIT S3 Documentation :  https://github.com/docker/docs/tree/main/content/storage  -- there is this : https://github.com/docker/docs/blob/5f94cb4f31535cef234fd52c5268660a7c4fe514/content/build/cache/backends/s3.md#L4 but not convinient_**
```

Comment:
https://github.com/docker/docs/blob/main/content/build/cache/backends/s3.md
^ also works, but I would defer to our docs maintained under https://zotregistry.dev